src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationFactory.cs

```diff
@@ -161,6 +164,17 @@ private void SetContentRoot(IWebHostBuilder builder)
             }
         }
 
+        private bool TrySetContentRootUsingEnvironmentVariables(IWebHostBuilder builder)
+        {
+            var expectedVariableName = "AspNetCore.Mvc.WebRoot-" + typeof(TEntryPoint).Assembly.GetName().Name;
+            var variableValue = Environment.GetEnvironmentVariable(expectedVariableName);
+            if (variableValue == null)
+                return false;
```

Comment:
use braces

Reply:
Seems fine to me.  Personally I can work with any env var names.  Adjust as you see fit :)